beacon-chain/main.go

```diff
@@ -86,6 +90,7 @@ func main() {
 			formatter := new(prefixed.TextFormatter)
 			formatter.TimestampFormat = "2006-01-02 15:04:05"
 			formatter.FullTimestamp = true
+			formatter.DisableColors = ctx.GlobalString(cmd.LogFileFormat.Name) != ""
```

Comment:
why do you disable color? with LogFileFormat.name

Reply:
The color are ANSI chars and when written to file - you get Gibberish. I didn't want to remove them via regex as this would be costly. @prestonvanloon instructed me to remove coloring in case the user wants persistent logs (when not writing log to file - you WILL get colored output in stdout).

Reply:
Please add a comment to the code to explain this condition

Reply:
Done